### DISCUSSION TITLE
Include subtasks when creating new issue from task

### DISCUSSION THREAD

**Discussion by mifriis** [community/community#4825] [Projects and Issues]
* [ ] Headline for the new issue
  * [ ] Some tasks
  * [ ] That would 
  * [ ] Be cool
  * [ ] To have added
  * [ ] To the new issue
  * [ ] Aswell

This would be a great addition to that feature.

**Comment by Narnach** [comment]
Came here to suggest that as well; now converting a large nested list into tasks involves copying the nested bullets to the new tasks manually. What we have is a great start, but still needs _more_ to feel natural.
